packages/cli/src/voting/getRewardsByRoundId.js

```diff
@@ -49,7 +49,7 @@ const getRewardsByRound = async (web3, votingContract, account) => {
       if (potentialRewards !== "0") {
         const resolvedVote = {
           price,
-          name: `${web3.utils.hexToUtf8(identifier)} @ ${style.formatSecondsToUtc(time)}`,
+          name: `${web3.utils.hexToUtf8(identifier)}@ ${style.formatSecondsToUtc(time)}`,
```

Comment:
Why remove the space here?

Reply:
Yeah, this one was a weird one. It triggered this rule: https://eslint.org/docs/rules/no-irregular-whitespace. Maybe the actual space was using a weird character. Let me try inserting back a normal space.

Reply:
Figured it wasn't a huge deal as it was just a slight change to the cli output, but probably worth trying to reinsert the space.

Reply:
Okay, yeah, somehow this was not a normal space character, but some other strange whitespace character (maybe a tab?). I updated it to a space, so this should be fixed?